backend/src/card_game/api.clj

```diff
@@ -18,27 +18,32 @@
 
 (defn get-game-as-player
   "Returns the part that ought to be visible to the player"
-  [game-state player]
+  [game-state player-id]
   {:game-id (:game-id game-state)
-   :player-id player
-   :hand (get-in game-state [:players (player-num game-state player) :hand])
+   :player-id player-id
+   :hand (get-in game-state [:players (player-num game-state player-id) :hand])
    :rows (mapv #(mapv (fn [card]
                         (assoc card :owner
-                          (translate-player game-state (:owner card) player)))
+                          (translate-player game-state (:owner card) player-id)))
                       %)
                (:rows game-state))
-   :winner (translate-player game-state (winner game-state) player)})
+   :winner (translate-player game-state (winner game-state) player-id)})
+
+(defn define-status
+  "Returns the status of the game"
+  [game-state player-id]
+  (nth ["Waiting for an opponent"
```

Comment:
I still don't like `(nth` with a `(dec (count`... any ideas on how to make it clearer?